CHANGES.md

```diff
@@ -10,6 +10,7 @@
 * Update `language-tags` to `0.3`.
 * `ServiceResponse::take_body`. [#2201]
 * `ServiceResponse::map_body` closure receives and returns `B` instead of `ResponseBody<B>` types. [#2201]
+* Change compression algorithm features flags. [#2250]
```

Comment:
Should be also included in `MIGRATION.md`?

Reply:
Agreed.

Reply:
Thanks, I miss this file. I just add a basic description on what should be change.
Feel free to review / update it.

Reply:
its not so important to add things to MIGRATION the mainainers will make sure it's compiled from the important changes from beta